tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -156,6 +156,13 @@ public TracerSettings(IConfigurationSource source)
                             // default value
                             8125;
 
+            var metricsUri = source?.GetString(ConfigurationKeys.MetricsAgentUri) ??
+                             // default value
+                             "http://localhost:9943";
```

Comment:
```suggestion
                             "http://localhost:9943/v2/datapoint";
```
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/078d6bd81584f8010d10e929584445a4f4e55f01/receiver/signalfxreceiver/receiver.go#L142
Probably also modification in our dev/docker-compose is needed.